concept/type/impl/ThingTypeImpl.java

```diff
@@ -153,8 +157,13 @@ public void unsetOwns(AttributeType attributeType) {
         if (getInstances().anyMatch(thing -> thing.getHas(attributeType).findAny().isPresent())) {
             throw exception(GraknException.of(INVALID_UNDEFINE_OWNS_HAS_INSTANCES, vertex.label(), attVertex.label()));
         }
-        if ((edge = vertex.outs().edge(OWNS, attVertex)) != null) edge.delete();
         if ((edge = vertex.outs().edge(OWNS_KEY, attVertex)) != null) edge.delete();
+        else if ((edge = vertex.outs().edge(OWNS, attVertex)) != null) edge.delete();
+        else if (this.getOwns().anyMatch(attr -> attr.equals(attributeType)))
+            throw exception(GraknException.of(INVALID_UNDEFINE_INHERITED_OWNS,
+                    this.getLabel().toString(), attributeType.getLabel().toString()));
+        else throw exception(GraknException.of(INVALID_UNDEFINE_NONEXISTENT_OWNS,
+                    this.getLabel().toString(), attributeType.getLabel().toString()));
```

Comment:
instead of taking no action when the edge is not found, we throw one of the two previously defined exceptions- inherited if inherited, nonexistent if unconnected.

Reply:
if it's multiline, we prefer to use the curly brackets, @Phillammon

Reply:
and are you sure the exception constructor don't fit one line (up to 120 char width) ?